### DISCUSSION TITLE
Is it possible to run this desktop environment on Ubuntu? I.e. modify the Ubuntu desktop environment for this project?

### DISCUSSION THREAD

**Discussion by ghost** [DustinBrett/daedalOS#195] [Q&A] [answered]
Hi all! I'm new to programming and I want to know if this desktop environment could be installed on Ubuntu. I would be happy if I could install this desktop environment on Ubuntu, is this possible? Is it possible to run this desktop environment on Ubuntu? I.e. modify the ubuntu desktop environment for this project?

**Comment by DustinBrett** [comment, marked as answer]
Hi @codehangen ,

Thanks for asking! It's not really a desktop environment in the traditional sense and so is not portable in that way. It basically needs to live inside a browser.
